doc/rados/troubleshooting/troubleshooting-mon.rst

```diff
@@ -127,10 +127,11 @@ and ``quorum_status``.
 Understanding mon_status
 ========================
 
-The status of the monitor (as reported by the ``ceph tell mon.X mon_status``
-command) can always be obtained via the admin socket. This command outputs a
-great deal of information about the monitor (including the information found in
-the output of the ``quorum_status`` command).
+The status of a Monitor (as reported by the ``ceph tell mon.X mon_status``
+command) can always be obtained via the admin socket. The ``ceph tell mon.X
```

Comment:
can be obtained via the daemon's admin socket.

Reply:
I like that better.

Accepted.